src/index.css

```diff
@@ -49,3 +49,7 @@ ul {
 .game-info {
   margin-left: 20px;
 }
+
+.winning-squares {
```

Comment:
Very minor suggestion but I would name this in the singular, i.e. `winning-square`.

Reply:
I like these suggestions.  I appreciate your attention to detail.  It helps me be the developer I would like to be! @jduffey